win32ss/user/ntuser/message.c

```diff
@@ -12,9 +12,33 @@
 DBG_DEFAULT_CHANNEL(UserMsg);
 
 #define PM_BADMSGFLAGS ~((QS_RAWINPUT << 16)|PM_QS_SENDMESSAGE|PM_QS_PAINT|PM_QS_POSTMESSAGE|PM_QS_INPUT|PM_NOYIELD|PM_REMOVE)
+#define EXCEPTION_ACCESS_VIOLATION ((DWORD)0xC0000005)
```

Comment:
```suggestion
```
This is called `STATUS_ACCESS_VIOLATION`

Reply:
Removed. Thanks.